index.js

```diff
@@ -30,13 +30,6 @@ module.exports = function imageResizer(options) {
     format      : null
   });
 
-  if (options.height == null && options.width) {
-    options.height = null;
-  }
-  if (options.width == null && options.height) {
-    options.width = null;
-  }
-
```

Comment:
Not related to this PR, but this code is not explicit and the tests pass without it.